bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/ProgramForm.java

```diff
@@ -161,8 +161,8 @@ public ProgramForm() {
                             new ColumnSpec(ColumnSpec.LEFT, Sizes.dluX(15), FormSpec.NO_GROW),
                             new ColumnSpec(ColumnSpec.CENTER, Sizes.dluX(56), FormSpec.NO_GROW),
                             FormFactory.UNRELATED_GAP_COLSPEC,
-                            FormFactory.DEFAULT_COLSPEC,
-                            new ColumnSpec(ColumnSpec.LEFT, Sizes.DLUX11, FormSpec.NO_GROW),
+                            new ColumnSpec(ColumnSpec.CENTER, Sizes.dluX(56), FormSpec.NO_GROW),
```

Comment:
This is the minimum time column, and should have been consistent with the other time columns; instead, this `DEFAULT_COLSPEC` was used which left-justified everything and made it look out of place since all the other times are centre-justified.

I suspect someone didn't maintain it properly along the way.